main.go

```diff
@@ -510,6 +510,16 @@ func _main(ctx context.Context) error {
 		// and signal the websocket hub to send progress updates to clients.
 		barLoad = make(chan *dbtypes.ProgressBarLoad, 2)
 		explore.BeginSyncStatusUpdates(barLoad)
+
+		// Fetch the latest blockdata which needed to update the agendas db while
+		// db sync is in progress.
+		d, _, err := collector.Collect()
```

Comment:
`bci, err :=  baseDB.BlockchainInfo()` would be preferable here since `Collect` does lots of unnecessary stuff here.